Makefile

```diff
@@ -60,6 +60,7 @@ dockerpush: ## Push the docker image to dockerhub
 	docker push $(TAGGED_REPO)
 
 chainlink: $(SGX_BUILD_ENCLAVE)
+	CGO_ENABLED=0 go run gui/main.go "${CURDIR}/store"
```

Comment:
@se3000 any opinions on just using the gui/main command for this? I figured I might as well just use that since it's already there.
I didn't bump the packr version as there were some backwards compatibility issues – I can work on that in a new PR.

Reply:
Also, did you want me to remove some of the contracts that aren't going to be used for now?

Reply:
Using the command from the gui package seems fine for now. Delaying the packr version bump works for me, no pressing need.

Ideally we would only include the ABIs of the contracts we use in the binary, there are a lot of extra contracts that get built just for the sake of testing.